File: dotnetcore/StarterBot/Entities/Player.cs
using System.Collections.Generic;
using StarterBot.Enums;

namespace StarterBot.Entities
{
    public class Player
    {
        public int Id { get; set; }
        public MapPosition Position { get; set; }
        public int Speed { get; set; }
        public State state { get; set; }
        public PowerUp[] PowerUps { get; set; }
        public bool Boosting { get; set; }
        public int BoostCounter { get; set; }
    }
}

File: starter-bots/python3/Player.py
class Player:
    def __init__(self, pid=None, speed=None, state=None, position=None, power_ups=None, boosting=None, boost_counter=None):
        self.id = pid
        self.position = position
        self.speed = speed
        self.state = state
        self.power_ups = power_ups
        self.boosting = boosting
        self.boost_counter = boost_counter
